Gemfile.lock

```diff
@@ -106,23 +106,23 @@ GEM
       aws-eventstream (~> 1, >= 1.0.2)
     base64 (0.2.0)
     bcrypt (3.1.20)
-    bigdecimal (3.1.7)
+    bigdecimal (3.1.8)
     bindex (0.8.1)
     bootsnap (1.17.0)
       msgpack (~> 1.2)
-    builder (3.2.4)
+    builder (3.3.0)
     byebug (11.1.3)
-    capybara (3.39.2)
+    capybara (3.40.0)
```

Comment:
manually updated capybara because they fixed the "uninitialized constant Rack::Handler (NameError)" issue (https://github.com/AllYourBot/hostedgpt/actions/runs/11415313307/job/31765228546)
see: 
https://discuss.rubyonrails.org/t/rails-7-1-2-upgrade-rack-handler-deprecation-warning/84166

and

https://github.com/teamcapybara/capybara/pull/2706